interactions/client/context.py

```diff
@@ -62,6 +62,8 @@ class _Context(DictSerializerMixin):
     deferred: bool = field(default=False)
 
     def __attrs_post_init__(self) -> None:
+        self.author = self.member
```

Comment:
```suggestion
        self.author = self.member or self.user
```

I think that'd make sense to have

Reply:
The old version didn't do that, nor is it documented to do that.

Reply:
Alright